The OpenAMP remote build is for wrong board

The OpenAMP sample does not build on the latest master.

Steps to reproduce;

1. git checkout b941561e # Check out a recent master
2. $ZEPHYR_BASE/scripts/sanitycheck -T $ZEPHYR_BASE/samples/subsys/ipc/openamp

```
-- Found PythonInterp: /usr/bin/python3 (found suitable version "3.5.2", minimum required is "3.4") 
-- Selected BOARD lpcxpresso54114_m4
Zephyr version: 1.13.99
Parsing Kconfig tree in /home/sebo/zephyr/Kconfig
Loading /home/sebo/zephyr/boards/arm/lpcxpresso54114_m4/lpcxpresso54114_m4_defconfig as base
Merging /home/sebo/zephyr/samples/subsys/ipc/openamp/prj.conf
-- Generating zephyr/include/generated/generated_dts_board.h
extract_dts_includes.py: Merge of 'category': 'required'  overwrites 'optional'.
-- Cache files will be written to: /home/sebo/.cache/zephyr
-- The C compiler identification is GNU 6.2.0
-- The CXX compiler identification is GNU 6.2.0
-- The ASM compiler identification is GNU
-- Found assembler: /home/sebo/zephyr-sdk/sysroots/x86_64-pokysdk-linux/usr/bin/arm-zephyr-eabi/arm-zephyr-eabi-gcc
-- Performing Test toolchain_is_ok
-- Performing Test toolchain_is_ok - Success
-- Host:    Linux/x86_64
-- Target:  Generic/arm
-- Machine: cortexm
-- C_FLAGS :  -Wall -Wextra
-- Looking for include file stdatomic.h
-- Looking for include file stdatomic.h - found
-- Looking for include file fcntl.h
-- Looking for include file fcntl.h - not found
-- Build type:  Debug
-- Host:    Linux/x86_64
-- Target:  Generic/arm
-- Machine: cortexm
-- Configuring done
-- Generating done
-- Build files have been written to: /home/sebo/zephyr/sanity-out/lpcxpresso54114_m4/samples/subsys/ipc/openamp/test
make[1]: Entering directory '/home/sebo/zephyr/sanity-out/lpcxpresso54114_m4/samples/subsys/ipc/openamp/test'
make[2]: Entering directory '/home/sebo/zephyr/sanity-out/lpcxpresso54114_m4/samples/subsys/ipc/openamp/test'
make[3]: Entering directory '/home/sebo/zephyr/sanity-out/lpcxpresso54114_m4/samples/subsys/ipc/openamp/test'
make[3]: Entering directory '/home/sebo/zephyr/sanity-out/lpcxpresso54114_m4/samples/subsys/ipc/openamp/test'
make[3]: Entering directory '/home/sebo/zephyr/sanity-out/lpcxpresso54114_m4/samples/subsys/ipc/openamp/test'
make[3]: Entering directory '/home/sebo/zephyr/sanity-out/lpcxpresso54114_m4/samples/subsys/ipc/openamp/test'
Scanning dependencies of target kobj_types_h_target
make[3]: Entering directory '/home/sebo/zephyr/sanity-out/lpcxpresso54114_m4/samples/subsys/ipc/openamp/test'
make[3]: Leaving directory '/home/sebo/zephyr/sanity-out/lpcxpresso54114_m4/samples/subsys/ipc/openamp/test'
make[3]: Entering directory '/home/sebo/zephyr/sanity-out/lpcxpresso54114_m4/samples/subsys/ipc/openamp/test'
Scanning dependencies of target openamp_remote
make[3]: Leaving directory '/home/sebo/zephyr/sanity-out/lpcxpresso54114_m4/samples/subsys/ipc/openamp/test'
[  1%] Generating include/generated/kobj-types-enum.h, include/generated/otype-to-str.h
Scanning dependencies of target syscall_macros_h_target
make[3]: Entering directory '/home/sebo/zephyr/sanity-out/lpcxpresso54114_m4/samples/subsys/ipc/openamp/test'
make[3]: Leaving directory '/home/sebo/zephyr/sanity-out/lpcxpresso54114_m4/samples/subsys/ipc/openamp/test'
Scanning dependencies of target driver_validation_h_target
make[3]: Leaving directory '/home/sebo/zephyr/sanity-out/lpcxpresso54114_m4/samples/subsys/ipc/openamp/test'
Scanning dependencies of target syscall_list_h_target
make[3]: Entering directory '/home/sebo/zephyr/sanity-out/lpcxpresso54114_m4/samples/subsys/ipc/openamp/test'
make[3]: Leaving directory '/home/sebo/zephyr/sanity-out/lpcxpresso54114_m4/samples/subsys/ipc/openamp/test'
[  1%] Creating directories for 'openamp_remote'
make[3]: Entering directory '/home/sebo/zephyr/sanity-out/lpcxpresso54114_m4/samples/subsys/ipc/openamp/test'
[  1%] Generating include/generated/syscall_macros.h
make[3]: Entering directory '/home/sebo/zephyr/sanity-out/lpcxpresso54114_m4/samples/subsys/ipc/openamp/test'
[  2%] Generating include/generated/driver-validation.h
[  2%] Generating misc/generated/syscalls.json
[  3%] No download step for 'openamp_remote'
[  3%] No patch step for 'openamp_remote'
[  4%] No update step for 'openamp_remote'
[  5%] Performing configure step for 'openamp_remote'
make[3]: Leaving directory '/home/sebo/zephyr/sanity-out/lpcxpresso54114_m4/samples/subsys/ipc/openamp/test'
[  5%] Built target syscall_macros_h_target
-- Found PythonInterp: /usr/bin/python3 (found suitable version "3.5.2", minimum required is "3.4") 
-- Selected BOARD nrf52840_pca10056
Zephyr version: 1.13.99
[  6%] Generating include/generated/syscall_dispatch.c, include/generated/syscall_list.h
make[3]: Leaving directory '/home/sebo/zephyr/sanity-out/lpcxpresso54114_m4/samples/subsys/ipc/openamp/test'
[  6%] Built target driver_validation_h_target
make[3]: Leaving directory '/home/sebo/zephyr/sanity-out/lpcxpresso54114_m4/samples/subsys/ipc/openamp/test'
[  6%] Built target kobj_types_h_target
make[3]: Leaving directory '/home/sebo/zephyr/sanity-out/lpcxpresso54114_m4/samples/subsys/ipc/openamp/test'
[  8%] Built target syscall_list_h_target
make[3]: Entering directory '/home/sebo/zephyr/sanity-out/lpcxpresso54114_m4/samples/subsys/ipc/openamp/test'
Scanning dependencies of target offsets
make[3]: Leaving directory '/home/sebo/zephyr/sanity-out/lpcxpresso54114_m4/samples/subsys/ipc/openamp/test'
make[3]: Entering directory '/home/sebo/zephyr/sanity-out/lpcxpresso54114_m4/samples/subsys/ipc/openamp/test'
[  9%] Building C object zephyr/CMakeFiles/offsets.dir/arch/arm/core/offsets/offsets.c.obj
[  9%] Linking C static library liboffsets.a
make[3]: Leaving directory '/home/sebo/zephyr/sanity-out/lpcxpresso54114_m4/samples/subsys/ipc/openamp/test'
[  9%] Built target offsets
make[3]: Entering directory '/home/sebo/zephyr/sanity-out/lpcxpresso54114_m4/samples/subsys/ipc/openamp/test'
Scanning dependencies of target offsets_h
make[3]: Leaving directory '/home/sebo/zephyr/sanity-out/lpcxpresso54114_m4/samples/subsys/ipc/openamp/test'
make[3]: Entering directory '/home/sebo/zephyr/sanity-out/lpcxpresso54114_m4/samples/subsys/ipc/openamp/test'
[ 10%] Generating include/generated/offsets.h
make[3]: Leaving directory '/home/sebo/zephyr/sanity-out/lpcxpresso54114_m4/samples/subsys/ipc/openamp/test'
[ 10%] Built target offsets_h
make[3]: Entering directory '/home/sebo/zephyr/sanity-out/lpcxpresso54114_m4/samples/subsys/ipc/openamp/test'
make[3]: Entering directory '/home/sebo/zephyr/sanity-out/lpcxpresso54114_m4/samples/subsys/ipc/openamp/test'
make[3]: Entering directory '/home/sebo/zephyr/sanity-out/lpcxpresso54114_m4/samples/subsys/ipc/openamp/test'
make[3]: Entering directory '/home/sebo/zephyr/sanity-out/lpcxpresso54114_m4/samples/subsys/ipc/openamp/test'
make[3]: Entering directory '/home/sebo/zephyr/sanity-out/lpcxpresso54114_m4/samples/subsys/ipc/openamp/test'
make[3]: Entering directory '/home/sebo/zephyr/sanity-out/lpcxpresso54114_m4/samples/subsys/ipc/openamp/test'
make[3]: Entering directory '/home/sebo/zephyr/sanity-out/lpcxpresso54114_m4/samples/subsys/ipc/openamp/test'
make[3]: Entering directory '/home/sebo/zephyr/sanity-out/lpcxpresso54114_m4/samples/subsys/ipc/openamp/test'
make[3]: Entering directory '/home/sebo/zephyr/sanity-out/lpcxpresso54114_m4/samples/subsys/ipc/openamp/test'
make[3]: Entering directory '/home/sebo/zephyr/sanity-out/lpcxpresso54114_m4/samples/subsys/ipc/openamp/test'
make[3]: Entering directory '/home/sebo/zephyr/sanity-out/lpcxpresso54114_m4/samples/subsys/ipc/openamp/test'
make[3]: Entering directory '/home/sebo/zephyr/sanity-out/lpcxpresso54114_m4/samples/subsys/ipc/openamp/test'
make[3]: Entering directory '/home/sebo/zephyr/sanity-out/lpcxpresso54114_m4/samples/subsys/ipc/openamp/test'
Scanning dependencies of target linker_script
make[3]: Leaving directory '/home/sebo/zephyr/sanity-out/lpcxpresso54114_m4/samples/subsys/ipc/openamp/test'
make[3]: Entering directory '/home/sebo/zephyr/sanity-out/lpcxpresso54114_m4/samples/subsys/ipc/openamp/test'
Scanning dependencies of target app
make[3]: Leaving directory '/home/sebo/zephyr/sanity-out/lpcxpresso54114_m4/samples/subsys/ipc/openamp/test'
make[3]: Entering directory '/home/sebo/zephyr/sanity-out/lpcxpresso54114_m4/samples/subsys/ipc/openamp/test'
[ 11%] Generating linker.cmd
Scanning dependencies of target lib__libc__minimal
make[3]: Leaving directory '/home/sebo/zephyr/sanity-out/lpcxpresso54114_m4/samples/subsys/ipc/openamp/test'
Scanning dependencies of target metal
make[3]: Leaving directory '/home/sebo/zephyr/sanity-out/lpcxpresso54114_m4/samples/subsys/ipc/openamp/test'
Scanning dependencies of target drivers__gpio
make[3]: Leaving directory '/home/sebo/zephyr/sanity-out/lpcxpresso54114_m4/samples/subsys/ipc/openamp/test'
[ 11%] Building C object CMakeFiles/app.dir/src/main_master.c.obj
[ 11%] Building C object CMakeFiles/app.dir/platform/platform_ops.c.obj
[ 12%] Building C object CMakeFiles/app.dir/platform/platform.c.obj
Scanning dependencies of target drivers__ipm
make[3]: Leaving directory '/home/sebo/zephyr/sanity-out/lpcxpresso54114_m4/samples/subsys/ipc/openamp/test'
make[3]: Entering directory '/home/sebo/zephyr/sanity-out/lpcxpresso54114_m4/samples/subsys/ipc/openamp/test'
Scanning dependencies of target boards__arm__lpcxpresso54114_m4
Scanning dependencies of target drivers__serial
make[3]: Leaving directory '/home/sebo/zephyr/sanity-out/lpcxpresso54114_m4/samples/subsys/ipc/openamp/test'
make[3]: Leaving directory '/home/sebo/zephyr/sanity-out/lpcxpresso54114_m4/samples/subsys/ipc/openamp/test'
make[3]: Entering directory '/home/sebo/zephyr/sanity-out/lpcxpresso54114_m4/samples/subsys/ipc/openamp/test'
Scanning dependencies of target arch__arm__core
make[3]: Leaving directory '/home/sebo/zephyr/sanity-out/lpcxpresso54114_m4/samples/subsys/ipc/openamp/test'
[ 12%] Building C object zephyr/drivers/serial/CMakeFiles/drivers__serial.dir/usart_mcux_lpc.c.obj
Scanning dependencies of target arch__arm__core__cortex_m
make[3]: Leaving directory '/home/sebo/zephyr/sanity-out/lpcxpresso54114_m4/samples/subsys/ipc/openamp/test'
make[3]: Entering directory '/home/sebo/zephyr/sanity-out/lpcxpresso54114_m4/samples/subsys/ipc/openamp/test'
Scanning dependencies of target zephyr
make[3]: Leaving directory '/home/sebo/zephyr/sanity-out/lpcxpresso54114_m4/samples/subsys/ipc/openamp/test'
make[3]: Entering directory '/home/sebo/zephyr/sanity-out/lpcxpresso54114_m4/samples/subsys/ipc/openamp/test'
make[3]: Entering directory '/home/sebo/zephyr/sanity-out/lpcxpresso54114_m4/samples/subsys/ipc/openamp/test'
make[3]: Leaving directory '/home/sebo/zephyr/sanity-out/lpcxpresso54114_m4/samples/subsys/ipc/openamp/test'
make[3]: Entering directory '/home/sebo/zephyr/sanity-out/lpcxpresso54114_m4/samples/subsys/ipc/openamp/test'
make[3]: Entering directory '/home/sebo/zephyr/sanity-out/lpcxpresso54114_m4/samples/subsys/ipc/openamp/test'
make[3]: Entering directory '/home/sebo/zephyr/sanity-out/lpcxpresso54114_m4/samples/subsys/ipc/openamp/test'
[ 12%] Building C object zephyr/lib/libc/minimal/CMakeFiles/lib__libc__minimal.dir/source/stdlib/atoi.c.obj
[ 12%] Building C object zephyr/ext/hal/libmetal/libmetal/lib/CMakeFiles/metal.dir/dma.c.obj
make[3]: Entering directory '/home/sebo/zephyr/sanity-out/lpcxpresso54114_m4/samples/subsys/ipc/openamp/test'
[ 13%] Building C object zephyr/drivers/ipm/CMakeFiles/drivers__ipm.dir/ipm_mcux.c.obj
[ 14%] Building C object zephyr/boards/arm/lpcxpresso54114_m4/CMakeFiles/boards__arm__lpcxpresso54114_m4.dir/pinmux.c.obj
Scanning dependencies of target kernel
make[3]: Leaving directory '/home/sebo/zephyr/sanity-out/lpcxpresso54114_m4/samples/subsys/ipc/openamp/test'
[ 15%] Building ASM object zephyr/arch/arm/core/cortex_m/CMakeFiles/arch__arm__core__cortex_m.dir/vector_table.S.obj
[ 15%] Building ASM object zephyr/arch/arm/core/CMakeFiles/arch__arm__core.dir/exc_exit.S.obj
[ 16%] Building C object zephyr/drivers/gpio/CMakeFiles/drivers__gpio.dir/gpio_mcux_lpc.c.obj
Scanning dependencies of target open_amp
make[3]: Leaving directory '/home/sebo/zephyr/sanity-out/lpcxpresso54114_m4/samples/subsys/ipc/openamp/test'
[ 16%] Built target linker_script
[ 17%] Building C object zephyr/CMakeFiles/zephyr.dir/arch/common/isr_tables.c.obj
[ 18%] Building C object zephyr/lib/libc/minimal/CMakeFiles/lib__libc__minimal.dir/source/stdlib/strtoul.c.obj
[ 19%] Building C object zephyr/lib/libc/minimal/CMakeFiles/lib__libc__minimal.dir/source/stdlib/strtol.c.obj
make[3]: Entering directory '/home/sebo/zephyr/sanity-out/lpcxpresso54114_m4/samples/subsys/ipc/openamp/test'
[ 20%] Building C object zephyr/CMakeFiles/zephyr.dir/lib/thread_entry.c.obj
make[3]: Entering directory '/home/sebo/zephyr/sanity-out/lpcxpresso54114_m4/samples/subsys/ipc/openamp/test'
[ 21%] Building C object zephyr/ext/lib/ipc/open-amp/open-amp/lib/CMakeFiles/open_amp.dir/common/hil.c.obj
[ 21%] Building C object zephyr/lib/libc/minimal/CMakeFiles/lib__libc__minimal.dir/source/stdlib/malloc.c.obj
[ 21%] Building ASM object zephyr/arch/arm/core/cortex_m/CMakeFiles/arch__arm__core__cortex_m.dir/reset.S.obj
[ 22%] Building C object zephyr/kernel/CMakeFiles/kernel.dir/alert.c.obj

warning: IPM_MCUX (defined at drivers/ipm/Kconfig:21) was assigned the value 'y' but got the value
'n'. You can check symbol information (including dependencies) in the 'menuconfig' interface (see
the Application Development Primer section of the manual), or in the Kconfig reference at
http://docs.zephyrproject.org/latest/reference/kconfig/CONFIG_IPM_MCUX.html (which is updated
regularly from the master branch). See the 'Setting configuration values' section of the Board
Porting Guide as well.

warning: PRINTK (defined at subsys/debug/Kconfig:63) was assigned the value 'n' but got the value
'y'. You can check symbol information (including dependencies) in the 'menuconfig' interface (see
the Application Development Primer section of the manual), or in the Kconfig reference at
http://docs.zephyrproject.org/latest/reference/kconfig/CONFIG_PRINTK.html (which is updated
regularly from the master branch). See the 'Setting configuration values' section of the Board
Porting Guide as well.
Parsing Kconfig tree in /home/sebo/zephyr/Kconfig
Loading /home/sebo/zephyr/boards/arm/nrf52840_pca10056/nrf52840_pca10056_defconfig as base
Merging /home/sebo/zephyr/samples/subsys/ipc/openamp/remote/prj.conf
[ 22%] Building C object zephyr/CMakeFiles/zephyr.dir/lib/crc/crc32_sw.c.obj
[ 23%] Building C object zephyr/arch/arm/core/CMakeFiles/arch__arm__core.dir/irq_init.c.obj
[ 24%] Linking C static library libdrivers__serial.a
[ 24%] Building C object zephyr/ext/lib/ipc/open-amp/open-amp/lib/CMakeFiles/open_amp.dir/common/sh_mem.c.obj
[ 25%] Building C object zephyr/ext/hal/libmetal/libmetal/lib/CMakeFiles/metal.dir/device.c.obj
[ 26%] Building ASM object zephyr/arch/arm/core/cortex_m/CMakeFiles/arch__arm__core__cortex_m.dir/nmi_on_reset.S.obj
[ 27%] Building C object CMakeFiles/app.dir/platform/resource_table.c.obj
make[3]: Leaving directory '/home/sebo/zephyr/sanity-out/lpcxpresso54114_m4/samples/subsys/ipc/openamp/test'
[ 28%] Building C object zephyr/arch/arm/core/cortex_m/CMakeFiles/arch__arm__core__cortex_m.dir/prep_c.c.obj
[ 29%] Building C object zephyr/CMakeFiles/zephyr.dir/lib/crc/crc16_sw.c.obj
[ 30%] Linking C static library libdrivers__ipm.a
[ 30%] Linking C static library libdrivers__gpio.a
[ 30%] Built target drivers__serial
[ 31%] Building C object zephyr/CMakeFiles/zephyr.dir/lib/crc/crc8_sw.c.obj
[ 31%] Linking C static library libboards__arm__lpcxpresso54114_m4.a
make[3]: Leaving directory '/home/sebo/zephyr/sanity-out/lpcxpresso54114_m4/samples/subsys/ipc/openamp/test'
[ 31%] Building C object zephyr/CMakeFiles/zephyr.dir/lib/mempool/mempool.c.obj
[ 31%] Built target drivers__ipm
[ 32%] Building C object zephyr/CMakeFiles/zephyr.dir/lib/rbtree/rb.c.obj
[ 32%] Building C object zephyr/arch/arm/core/cortex_m/CMakeFiles/arch__arm__core__cortex_m.dir/scb.c.obj
-- Generating zephyr/include/generated/generated_dts_board.h
[ 33%] Building C object zephyr/kernel/CMakeFiles/kernel.dir/device.c.obj
make[3]: Leaving directory '/home/sebo/zephyr/sanity-out/lpcxpresso54114_m4/samples/subsys/ipc/openamp/test'
make[3]: Leaving directory '/home/sebo/zephyr/sanity-out/lpcxpresso54114_m4/samples/subsys/ipc/openamp/test'
[ 33%] Building C object zephyr/kernel/CMakeFiles/kernel.dir/errno.c.obj
[ 34%] Building C object zephyr/lib/libc/minimal/CMakeFiles/lib__libc__minimal.dir/source/string/strncasecmp.c.obj
[ 34%] Built target boards__arm__lpcxpresso54114_m4
[ 35%] Building C object zephyr/arch/arm/core/CMakeFiles/arch__arm__core.dir/swap.c.obj
[ 35%] Built target drivers__gpio
[ 36%] Building C object zephyr/arch/arm/core/cortex_m/CMakeFiles/arch__arm__core__cortex_m.dir/nmi.c.obj
[ 37%] Building C object zephyr/kernel/CMakeFiles/kernel.dir/init.c.obj
[ 38%] Building C object zephyr/kernel/CMakeFiles/kernel.dir/idle.c.obj
[ 38%] Building ASM object zephyr/arch/arm/core/CMakeFiles/arch__arm__core.dir/swap_helper.S.obj
[ 39%] Building C object zephyr/arch/arm/core/cortex_m/CMakeFiles/arch__arm__core__cortex_m.dir/exc_manage.c.obj
[ 40%] Building C object zephyr/ext/lib/ipc/open-amp/open-amp/lib/CMakeFiles/open_amp.dir/common/firmware.c.obj
[ 41%] Linking C static library libapp.a
[ 42%] Building C object zephyr/lib/libc/minimal/CMakeFiles/lib__libc__minimal.dir/source/string/strstr.c.obj
[ 42%] Building C object zephyr/kernel/CMakeFiles/kernel.dir/mailbox.c.obj
[ 43%] Building C object zephyr/ext/hal/libmetal/libmetal/lib/CMakeFiles/metal.dir/init.c.obj
[ 44%] Building C object zephyr/ext/lib/ipc/open-amp/open-amp/lib/CMakeFiles/open_amp.dir/virtio/virtio.c.obj
[ 44%] Building C object zephyr/lib/libc/minimal/CMakeFiles/lib__libc__minimal.dir/source/string/string.c.obj
make[3]: Leaving directory '/home/sebo/zephyr/sanity-out/lpcxpresso54114_m4/samples/subsys/ipc/openamp/test'
[ 44%] Building C object zephyr/ext/hal/libmetal/libmetal/lib/CMakeFiles/metal.dir/io.c.obj
[ 44%] Building C object zephyr/ext/lib/ipc/open-amp/open-amp/lib/CMakeFiles/open_amp.dir/virtio/virtqueue.c.obj
[ 45%] Building C object zephyr/arch/arm/core/CMakeFiles/arch__arm__core.dir/fault.c.obj
[ 45%] Built target app
[ 46%] Building C object zephyr/kernel/CMakeFiles/kernel.dir/mem_slab.c.obj
[ 47%] Building C object zephyr/lib/libc/minimal/CMakeFiles/lib__libc__minimal.dir/source/stdout/prf.c.obj
[ 48%] Building C object zephyr/lib/libc/minimal/CMakeFiles/lib__libc__minimal.dir/source/stdout/sprintf.c.obj
[ 50%] Building C object zephyr/lib/libc/minimal/CMakeFiles/lib__libc__minimal.dir/source/stdout/fprintf.c.obj
[ 51%] Building C object zephyr/kernel/CMakeFiles/kernel.dir/mempool.c.obj
[ 51%] Building C object zephyr/lib/libc/minimal/CMakeFiles/lib__libc__minimal.dir/source/stdout/stdout_console.c.obj
[ 51%] Building C object zephyr/kernel/CMakeFiles/kernel.dir/msg_q.c.obj
[ 51%] Linking C static library libarch__arm__core__cortex_m.a
[ 52%] Building C object zephyr/CMakeFiles/zephyr.dir/misc/printk.c.obj
[ 53%] Building C object zephyr/ext/lib/ipc/open-amp/open-amp/lib/CMakeFiles/open_amp.dir/rpmsg/remote_device.c.obj
[ 54%] Building C object zephyr/arch/arm/core/CMakeFiles/arch__arm__core.dir/irq_manage.c.obj
[ 54%] Linking C static library liblib__libc__minimal.a
[ 55%] Building C object zephyr/kernel/CMakeFiles/kernel.dir/mutex.c.obj
make[3]: Leaving directory '/home/sebo/zephyr/sanity-out/lpcxpresso54114_m4/samples/subsys/ipc/openamp/test'
[ 55%] Building C object zephyr/CMakeFiles/zephyr.dir/misc/generated/configs.c.obj
[ 55%] Built target arch__arm__core__cortex_m
[ 56%] Building C object zephyr/kernel/CMakeFiles/kernel.dir/pipes.c.obj
[ 56%] Building C object zephyr/kernel/CMakeFiles/kernel.dir/queue.c.obj
make[3]: Leaving directory '/home/sebo/zephyr/sanity-out/lpcxpresso54114_m4/samples/subsys/ipc/openamp/test'
[ 57%] Building C object zephyr/CMakeFiles/zephyr.dir/ext/hal/nxp/mcux/devices/LPC54114/fsl_clock.c.obj
[ 58%] Building C object zephyr/CMakeFiles/zephyr.dir/ext/hal/nxp/mcux/drivers/lpc/fsl_usart.c.obj
[ 58%] Built target lib__libc__minimal
[ 59%] Building C object zephyr/ext/hal/libmetal/libmetal/lib/CMakeFiles/metal.dir/log.c.obj
[ 60%] Building ASM object zephyr/CMakeFiles/zephyr.dir/ext/hal/nxp/mcux/devices/LPC54114/gcc/startup_LPC54114_cm4.S.obj
-- Cache files will be written to: /home/sebo/.cache/zephyr
[ 60%] Building C object zephyr/CMakeFiles/zephyr.dir/ext/hal/nxp/mcux/drivers/lpc/fsl_flexcomm.c.obj
[ 61%] Building C object zephyr/kernel/CMakeFiles/kernel.dir/sched.c.obj
[ 62%] Building C object zephyr/ext/hal/libmetal/libmetal/lib/CMakeFiles/metal.dir/shmem.c.obj
[ 63%] Building C object zephyr/CMakeFiles/zephyr.dir/drivers/console/uart_console.c.obj
[ 63%] Building C object zephyr/ext/hal/libmetal/libmetal/lib/CMakeFiles/metal.dir/version.c.obj
[ 63%] Building C object zephyr/CMakeFiles/zephyr.dir/drivers/pinmux/pinmux_mcux_lpc.c.obj
[ 63%] Building C object zephyr/arch/arm/core/CMakeFiles/arch__arm__core.dir/thread.c.obj
[ 64%] Building ASM object zephyr/arch/arm/core/CMakeFiles/arch__arm__core.dir/cpu_idle.S.obj
[ 65%] Building C object zephyr/kernel/CMakeFiles/kernel.dir/sem.c.obj
[ 67%] Building C object zephyr/ext/lib/ipc/open-amp/open-amp/lib/CMakeFiles/open_amp.dir/rpmsg/rpmsg.c.obj
[ 67%] Building C object zephyr/ext/hal/libmetal/libmetal/lib/CMakeFiles/metal.dir/system/zephyr/alloc.c.obj
[ 68%] Building C object zephyr/CMakeFiles/zephyr.dir/drivers/timer/sys_clock_init.c.obj
[ 68%] Building C object zephyr/kernel/CMakeFiles/kernel.dir/stack.c.obj
[ 69%] Building C object zephyr/kernel/CMakeFiles/kernel.dir/sys_clock.c.obj
-- The C compiler identification is GNU 6.2.0
[ 69%] Building C object zephyr/ext/lib/ipc/open-amp/open-amp/lib/CMakeFiles/open_amp.dir/rpmsg/rpmsg_core.c.obj
[ 70%] Building ASM object zephyr/arch/arm/core/CMakeFiles/arch__arm__core.dir/fault_s.S.obj
[ 71%] Building C object zephyr/ext/hal/libmetal/libmetal/lib/CMakeFiles/metal.dir/system/zephyr/condition.c.obj
-- The CXX compiler identification is GNU 6.2.0
-- The ASM compiler identification is GNU
-- Found assembler: /home/sebo/zephyr-sdk/sysroots/x86_64-pokysdk-linux/usr/bin/arm-zephyr-eabi/arm-zephyr-eabi-gcc
-- Performing Test toolchain_is_ok
[ 72%] Building C object zephyr/CMakeFiles/zephyr.dir/drivers/timer/cortex_m_systick.c.obj
[ 73%] Building C object zephyr/kernel/CMakeFiles/kernel.dir/thread_abort.c.obj
[ 73%] Building C object zephyr/arch/arm/core/CMakeFiles/arch__arm__core.dir/fatal.c.obj
[ 74%] Building C object zephyr/kernel/CMakeFiles/kernel.dir/system_work_q.c.obj
[ 74%] Building C object zephyr/ext/hal/libmetal/libmetal/lib/CMakeFiles/metal.dir/system/zephyr/device.c.obj
[ 75%] Building C object zephyr/arch/arm/core/CMakeFiles/arch__arm__core.dir/sys_fatal_error_handler.c.obj
[ 76%] Building C object zephyr/kernel/CMakeFiles/kernel.dir/version.c.obj
[ 76%] Building C object zephyr/kernel/CMakeFiles/kernel.dir/thread.c.obj
[ 76%] Building C object zephyr/kernel/CMakeFiles/kernel.dir/work_q.c.obj
[ 77%] Building C object zephyr/kernel/CMakeFiles/kernel.dir/smp.c.obj
[ 78%] Building C object zephyr/ext/lib/ipc/open-amp/open-amp/lib/CMakeFiles/open_amp.dir/remoteproc/elf_loader.c.obj
[ 79%] Building C object zephyr/kernel/CMakeFiles/kernel.dir/timer.c.obj
[ 80%] Building C object zephyr/arch/arm/core/CMakeFiles/arch__arm__core.dir/thread_abort.c.obj
[ 81%] Building C object zephyr/ext/hal/libmetal/libmetal/lib/CMakeFiles/metal.dir/system/zephyr/init.c.obj
[ 82%] Building C object zephyr/ext/hal/libmetal/libmetal/lib/CMakeFiles/metal.dir/system/zephyr/irq.c.obj
[ 82%] Linking C static library libzephyr.a
-- Performing Test toolchain_is_ok - Success
[ 82%] Building ASM object zephyr/arch/arm/core/CMakeFiles/arch__arm__core.dir/isr_wrapper.S.obj
[ 82%] Building C object zephyr/ext/hal/libmetal/libmetal/lib/CMakeFiles/metal.dir/system/zephyr/log.c.obj
[ 83%] Building C object zephyr/ext/hal/libmetal/libmetal/lib/CMakeFiles/metal.dir/system/zephyr/shmem.c.obj
make[3]: Leaving directory '/home/sebo/zephyr/sanity-out/lpcxpresso54114_m4/samples/subsys/ipc/openamp/test'
[ 84%] Building C object zephyr/ext/lib/ipc/open-amp/open-amp/lib/CMakeFiles/open_amp.dir/remoteproc/remoteproc.c.obj
[ 84%] Built target zephyr
[ 85%] Building C object zephyr/ext/hal/libmetal/libmetal/lib/CMakeFiles/metal.dir/system/zephyr/time.c.obj
[ 85%] Building C object zephyr/ext/hal/libmetal/libmetal/lib/CMakeFiles/metal.dir/system/zephyr/cortexm/sys.c.obj
[ 86%] Building C object zephyr/ext/lib/ipc/open-amp/open-amp/lib/CMakeFiles/open_amp.dir/remoteproc/rsc_table_parser.c.obj
[ 86%] Building C object zephyr/ext/lib/ipc/open-amp/open-amp/lib/CMakeFiles/open_amp.dir/remoteproc/remoteproc_loader.c.obj
[ 87%] Linking C static library libarch__arm__core.a
[ 88%] Linking C static library libmetal.a
make[3]: Leaving directory '/home/sebo/zephyr/sanity-out/lpcxpresso54114_m4/samples/subsys/ipc/openamp/test'
make[3]: Leaving directory '/home/sebo/zephyr/sanity-out/lpcxpresso54114_m4/samples/subsys/ipc/openamp/test'
[ 88%] Built target arch__arm__core
[ 88%] Built target metal
[ 88%] Linking C static library libkernel.a
make[3]: Leaving directory '/home/sebo/zephyr/sanity-out/lpcxpresso54114_m4/samples/subsys/ipc/openamp/test'
[ 88%] Built target kernel
-- Host:    Linux/x86_64
-- Target:  Generic/arm
-- Machine: cortexm
-- C_FLAGS :  -Wall -Wextra
-- Looking for include file stdatomic.h
[ 88%] Linking C static library libopen_amp.a
make[3]: Leaving directory '/home/sebo/zephyr/sanity-out/lpcxpresso54114_m4/samples/subsys/ipc/openamp/test'
[ 88%] Built target open_amp
-- Looking for include file stdatomic.h - found
-- Looking for include file fcntl.h
-- Looking for include file fcntl.h - not found
-- Build type:  Debug
-- Host:    Linux/x86_64
-- Target:  Generic/arm
-- Machine: cortexm
CMake Error at ../../../../../CMakeLists.txt:537 (message):
  The Zephyr library 'drivers__ipm' was created without source files.  Empty
  (non-imported) libraries are not supported.  Either make sure that the
  library has the sources it should have, or make sure it is not created when
  it has no source files.


-- Configuring incomplete, errors occurred!
See also "/home/sebo/zephyr/sanity-out/lpcxpresso54114_m4/samples/subsys/ipc/openamp/test/openamp_remote-prefix/src/openamp_remote-build/CMakeFiles/CMakeOutput.log".
See also "/home/sebo/zephyr/sanity-out/lpcxpresso54114_m4/samples/subsys/ipc/openamp/test/openamp_remote-prefix/src/openamp_remote-build/CMakeFiles/CMakeError.log".
CMakeFiles/openamp_remote.dir/build.make:105: recipe for target 'openamp_remote-prefix/src/openamp_remote-stamp/openamp_remote-configure' failed
make[3]: *** [openamp_remote-prefix/src/openamp_remote-stamp/openamp_remote-configure] Error 1
make[3]: Target 'CMakeFiles/openamp_remote.dir/build' not remade because of errors.
make[3]: Leaving directory '/home/sebo/zephyr/sanity-out/lpcxpresso54114_m4/samples/subsys/ipc/openamp/test'
CMakeFiles/Makefile2:168: recipe for target 'CMakeFiles/openamp_remote.dir/all' failed
make[2]: *** [CMakeFiles/openamp_remote.dir/all] Error 2
make[2]: Target 'all' not remade because of errors.
make[2]: Leaving directory '/home/sebo/zephyr/sanity-out/lpcxpresso54114_m4/samples/subsys/ipc/openamp/test'
Makefile:129: recipe for target 'all' failed
make[1]: *** [all] Error 2
make[1]: Target 'default_target' not remade because of errors.
make[1]: Leaving directory '/home/sebo/zephyr/sanity-out/lpcxpresso54114_m4/samples/subsys/ipc/openamp/test'

```

--- a/samples/subsys/ipc/ipm_mcux/CMakeLists.txt
+++ b/samples/subsys/ipc/ipm_mcux/CMakeLists.txt
@@ -14,6 +14,8 @@ ExternalProject_Add(
   SOURCE_DIR ${APPLICATION_SOURCE_DIR}/remote
   INSTALL_COMMAND ""      # This particular build system has no install command
   BUILD_BYPRODUCTS "${CMAKE_CURRENT_BINARY_DIR}/ipm_mcux_remote-prefix/src/ipm_mcux_remote-build/zephyr/zephyr.bin"
+  CMAKE_CACHE_ARGS -DBOARD:STRING=${BOARD}
+  # NB: Do we need to pass on more CMake variables?
 )
 
 project(NONE)

--- a/samples/subsys/ipc/openamp/CMakeLists.txt
+++ b/samples/subsys/ipc/openamp/CMakeLists.txt
@@ -25,6 +25,8 @@ ExternalProject_Add(
   SOURCE_DIR ${APPLICATION_SOURCE_DIR}/remote
   INSTALL_COMMAND ""      # This particular build system has no install command
   BUILD_BYPRODUCTS "${CMAKE_CURRENT_BINARY_DIR}/openamp_remote-prefix/src/openamp_remote-build/zephyr/zephyr.bin"
+  CMAKE_CACHE_ARGS -DBOARD:STRING=${BOARD}
+  # NB: Do we need to pass on more CMake variables?
 )
 add_dependencies(core_m0_inc_target openamp_remote)
 

